src/cui.cpp

```diff
@@ -277,6 +283,18 @@ int GreatestFirst(const void *ma, const void *mb) {
   return 1;
 }
 
+int FrozenOrder(const void *ma, const void *mb) {
+  Line *a = *(Line **)ma;
+  Line *b = *(Line **)mb;
+  auto ita = std::find(frozenPids.begin(), frozenPids.end(), a->get_pid());
+  auto itb = std::find(frozenPids.begin(), frozenPids.end(), b->get_pid());
+  int ia = (ita == frozenPids.end()) ? INT_MAX : std::distance(frozenPids.begin(), ita);
+  int ib = (itb == frozenPids.end()) ? INT_MAX : std::distance(frozenPids.begin(), itb);
+  if (ia != ib)
+    return ia - ib;
+  return GreatestFirst(ma, mb);
```

Comment:
It looks like this means after freezing, the 'frozen' pids stay in a stable order, but any new pids will still be sorted on each refresh. That seems surprising to me? Perhaps we should update the list of frozen pids after each 'frozen refresh'?